cves/CVE-2013-1665.yml

```diff
@@ -159,8 +183,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    A blog from Python on this vulnerability indicates that the developers have
+    to re-implement the XML specifications properly, which indicates that there's
+    a violation of XML specifications.
+  answer: true
```

Comment:
Make sure to include the link to said blog, or and add maybe quotes to it. Suggest using APA format for references

Reply:
Yup, I will put in the link of the blog